queries.sql

```diff
@@ -1,3 +1,90 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+/* Find all animals whose name ends in "mon". */
+SELECT * FROM animals WHERE name LIKE '%mon%';
+/* List the name of all animals born between 2016 and 2019. */
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+/* List the name of all animals that are neutered and have less than 3 escape attempts.*/
+SELECT * FROM animals WHERE neutered = TRUE AND escape_attempt < 3;
+/* List the date of birth of all animals named either "Agumon" or "Pikachu" */
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+/* List name and escape attempts of animals that weigh more than 10.5kg */
+SELECT name, escape_attempt FROM animals WHERE weight_kg >10.5;
+/* Find all animals that are neutered. */
+SELECT * FROM animals WHERE neutered = TRUE;
+/* Find all animals not named Gabumon. */
+ SELECT * FROM animals WHERE name <> 'Gabumons';
+/* Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg) */
+ SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+ -- TRANSACTION FOR UPDATING SPECIES COLUMN WITH UNSPECIFIED and ROLLBACK
+
+BEGIN;
+
+UPDATE animals SET species = 'unspecified';
+
+SELECT * FROM animals; 
+
+ROLLBACK; 
+
+SELECT * FROM animals;
+
+--  TRANSACTION FOR UPDATING TO DIGIMON AND POKEMON WITH COMMIT
+
+BEGIN; 
+
+UPDATE animals SET species = 'digimon' WHERE NAME LIKE '%mon%';
```

Comment:
- [ ] It would be better if we change the `%mon%` to `%mon` as in the screenshot below:
![Screenshot from 2022-08-11 10-48-48](https://user-images.githubusercontent.com/46388113/184117860-57040afa-6e98-4c0a-8bdf-53335a030332.png)